frontend/packages/dev-console/src/components/import/app/AppSection.tsx

```diff
@@ -21,6 +22,7 @@ const AppSection: React.FC<AppSectionProps> = ({ project }) => {
       <ApplicationSelector namespace={project.name} />
       <InputField
         type="text"
+        data-test-id="application-form-app-name"
```

Comment:
@christianvogt You mentioned using `'data-test-id'` instead of `testId`. Whereas @rohitkrai03 is asking for me to use `testId`. I currently have it as `testId`. Can you explain what you want and why?

Reply:
I tried this change but could not access it from the DOM, so I went back to `data-test-id`.